frontend/src/components/project/ProjectOverview.js

```diff
@@ -41,6 +42,16 @@ const useStyles = makeStyles((theme) => ({
     marginRight: theme.spacing(1),
     color: theme.palette.primary.main,
   },
+  largeScreenHeader: {
+    paddingTop: theme.spacing(4),
+    paddingBottom: theme.spacing(4),
+    textAlign: "center",
+  },
+  goBackButtonContainer: {
+    marginLeft: theme.spacing(1),
+    marginTop: theme.spacing(1),
```

Comment:
Is there a way to position the button on the image without using `absolute`?

Reply:
That would only be possible if you use a div with the background property rather than an image. If you'd like to use this option I can explain it more in detail in a call :)